src/modules/Edition/index.ts

```diff
@@ -3,18 +3,31 @@ import { ethers, BigNumber, Contract } from "ethers";
 import DCNT721A from '../../contracts/DCNT721A.json';
 import { MetadataRendererInit } from '../MetadataRenderer';
 
+export type TokenGateConfig = {
+  tokenAddress: string;
+  minBalance: number;
+  saleType: number;
+}
+
 const deploy = async (
   sdk: SDK,
   name: string,
   symbol: string,
+  adjustableCap: boolean,
```

Comment:
carry over from contracts PR. This is a new variable and current name implies `uint256` but this is actually a `bool`. I suggest 
```suggestion
  isAdjustableCap: boolean,
```